markdown/usage/installation.md

```diff
@@ -31,125 +17,60 @@ java -version
 # Install Nextflow
 curl -fsSL get.nextflow.io | bash
 
-# Add Nextflow binary to your PATH:
+# Add Nextflow binary to your user's PATH:
 mv nextflow ~/bin/
 # OR system-wide installation:
 # sudo mv nextflow /usr/local/bin
 ```
 
-See [nextflow.io](https://www.nextflow.io/) for further instructions on how to install and configure Nextflow.
-
-## Install the pipeline
-
-### Automatic
-This pipeline itself needs no installation - nextflow will automatically fetch it from GitHub if `nf-core/YOUR_PIPELINE` is specified as the pipeline name.
-
-### Offline
-The above method requires an internet connection so that Nextflow can download the pipeline files. If you're running on a system that has no internet connection, you'll need to download and transfer the pipeline files manually:
-
-```bash
-wget https://github.com/nf-core/YOUR_PIPELINE/archive/master.zip
-mkdir -p ~/my-pipelines/nf-core/
-unzip master.zip -d ~/my-pipelines/nf-core/
-cd ~/my_data/
-nextflow run ~/my-pipelines/nf-core/YOUR_PIPELINE-master
-```
-
-To stop nextflow from looking for updates online, you can tell it to run in offline mode by specifying the following environment variable in your ~/.bashrc file:
+You can also install Nextflow using [Bioconda](https://bioconda.github.io/):
 
 ```bash
-export NXF_OFFLINE='TRUE'
+conda install -c bioconda nextflow
 ```
 
-### Development
-
-If you would like to make changes to the pipeline, it's best to make a fork on GitHub and then clone the files. Once cloned you can run the pipeline directly as above.
-
-
-## Pipeline configuration
-By default, the pipeline loads a basic resource configuration:
-
-```nextflow
-process {
-
-  cpus = { check_max( 2, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
-
-  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'terminate' }
-  maxRetries = 1
-  maxErrors = '-1'
-
-  // Process-specific resource requirements
-  withName: fastqc {
-    time = { check_max( 8.h * task.attempt, 'time' ) }
-  }
-  withName:toolX {
-    cpus = { check_max( 8, 'cpus' ) }
-    memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  }
-  withName: toolY {
-    cpus = { check_max( 10, 'cpus' ) }
-    memory = { check_max( 200.GB * task.attempt, 'memory' ) }
-    time = { check_max( 5.h * task.attempt, 'time' ) }
-  }
-}
-
-params {
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-}
-```
+We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation. This makes it easier to update.
 
-This uses a number of sensible defaults for process requirements and is suitable for running
-on a simple (if powerful!) local server.
+Updating nextflow is as simple as running `nextflow self-update`.
```

Comment:
If you specify how to install with condat, should we specify how to do a Nextflow update with conda as well?

Reply:
Yeah, I guess so. Does `nextflow self-update` not work with conda? Probably not I guess..

Reply:
Should we just drop the conda install command and link the bioconda nextflow page?

Reply:
I started doing this but then we still have the problem with the self-update command. Instead I've just added mention of `conda update nextflow`..